src/2ls/2ls_parse_options.h

```diff
@@ -59,32 +59,28 @@ class optionst;
   "(preconditions)(sufficient)" \
   "(show-locs)(show-vcc)(show-properties)(trace)(show-stats)" \
   "(show-goto-functions)(show-guards)(show-defs)(show-ssa)(show-assignments)" \
-  "(show-invariants)(std-invariants)" \
+  "(show-invariants)(std-invariants)(show-symbol-table)" \
   "(property):(all-properties)(k-induction)(incremental-bmc)" \
   "(no-spurious-check)(all-functions)" \
   "(no-simplify)(no-fixed-point)" \
   "(graphml-witness):(json-cex):" \
   "(no-spurious-check)(stop-on-fail)" \
   "(competition-mode)(slice)(no-propagation)(independent-properties)" \
-  "(constant-propagation)" \
   "(no-unwinding-assertions)"
   // the last line is for CBMC-regression testing only
 
 class twols_parse_optionst:
   public parse_options_baset,
-  public language_uit
+  public messaget
```

Comment:
💡 Preferably add a `messaget log` member than deriving from `messaget`.

Reply:
I wanted to follow CBMC's example (`cbmc_parse_optionst` used to inherit from `language_uit` and now inherits from `messaget`). Moving from inheritance to a member would require a lot of changes throughout the whole module, so I'd rather keep it as is for now to keep the changes minimal.

Reply:
Ok!